watchtower-plugin/src/net/http.rs

```diff
@@ -372,9 +475,13 @@ mod tests {
         let (tower_sk, tower_pk) = cryptography::get_random_keypair();
```

Comment:
The exact same comments from the last test apply to this one, it has the exact same calls and assignemnts.